client/app/styles/main.less

```diff
@@ -958,6 +947,25 @@ h2 {
   }
 }
 
+.node-control-button {
+  .btn-opacity;
+  padding: 6px;
+  margin-left: 2px;
+  font-size: 110%;
```

Comment:
Do you prefer `em` or `%`? Might be nice to settle on one. `lesshint` (less linting could be interesting, I love our JS linting).

I prefer `em` I think.